src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -494,6 +494,9 @@ function ReportActionItemMessageEdit(
                             style={[styles.textInputCompose, styles.flex1, styles.bgTransparent]}
                             onFocus={() => {
                                 setIsFocused(true);
+                                if (textInputRef.current) {
+                                    ReportActionComposeFocusManager.editComposerRef.current = textInputRef.current;
```

Comment:
Let us reset the `editComposerRef` when the focus is lost to avoid dangling references. One way to do this is by resetting it inside `ReportActionComposeFocusManager.clear`.

Reply:
Maybe I am missing something here but why would we need `editComposerRef` after `ReportActionComposeFocusManager.clear` is called [here](https://github.com/Expensify/App/blob/fd16fdde825ac626d76f113dfff8d5bbfece32b8/src/pages/home/report/ReportActionItemMessageEdit.tsx#L274)?

Reply:
> when ReportActionComposeFocusManager.clear we can reset `editComposerRef`

Yes, that's what I meant

Reply:
Can you please check if we also have to set the `editComposerRef` on component mount of `ReportActionItemMessageEdit` because I ran into issues where the edit composer did not blur as `editComposerRef` was not set at all? Here is a video that demonstrates this. Please have a look.

https://github.com/user-attachments/assets/f0f61265-e7b6-4875-a1fb-63023b894610